plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: palo_alto_cortex_xdr
 title: Palo Alto Cortex XDR
 description: Stop modern attacks with the industry's first extended detection and response platform that spans your endpoints, network and cloud data
-version: 1.0.0
+version: 1.0.1
```

```suggestion
version: 1.1.0
```
Minor version bump since you've added actions